lucille-core/src/main/java/com/kmwllc/lucille/core/Indexer.java

```diff
@@ -48,10 +49,11 @@ public void terminate() {
   public Indexer(Config config, IndexerMessageManager manager, String metricsPrefix) {
     this.manager = manager;
     this.idOverrideField = config.hasPath("indexer.idOverrideField") ? config.getString("indexer.idOverrideField") : null;
+    this.indexOverrideField = config.hasPath("indexer.indexOverrideField") ? config.getString("indexer.indexOverrideField") : null;
```

Comment:
What happens if someone specifies indexer.indexOverrideField with an elasticsearch or opensearch indexer? Seems like everything would work, but we'd be creating a MultiBatch without actually sending docs to separate indices. Do we plan to support this property for elastic and opensearch or is it intended as solr-only? Should probably throw an exception if someone tries to use it in a context where it's not supported.

Reply:
@rseitz I just added exceptions to the ElasticIndexer, OpenSearchIndexer, and CSVIndexer. Please take a look.

I do think it makes sense to eventually support this in the other indexers, but this LC-273 was for supporting it in Solr.

Reply:
The current approach forces every Indexer subclass that doesn't support indexer.OverrideField to check for it and throw an exception.

It might be better to have a design where the exception is thrown by default unless the subclass does something special to support this property. However, if we're planning to support this property in all the other Indexer subclasses, the point might be moot.

So basically, I'm OK with what's there now but wanted to mention this for you to consider.